src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -347,7 +342,7 @@ export const CloudEnvironmentModal = ({
             disabled: isCloudEnvForToolDisabled,
             onClick: () => setViewMode(toolLabel)
           }, [
-            img({ src: cloudIcon, style: { height: 20, width: 20, opacity: isCloudEnvForToolDisabled ? 0.4 : 1 } }), //TODO: why doesn't this icon disable properly color-wise?
```

Comment:
What does it mean for an icon to "disable properly colour-wise"?

Reply:
Oh, originally the img was misbehaving and was not being grayed out when the disabled property was true (it was staying the same color)